style.css

```diff
@@ -0,0 +1,198 @@
+body {
+  font-family: Poppins, sans-serif;
+  padding: 22px;
+}
+
+.container {
+  width: 375px;
+  height: 718px;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid black;
```

Comment:
- [ ] Good job making almost all designs from the previous reviewer, however, your changes seem to have made a different viewpoint. kindly fix it in line with the responsiveness. An an excellent design you made there 👏 

## New design
![image](https://user-images.githubusercontent.com/50990925/211991776-e04d3aeb-fb79-440b-bdb2-62bd590dfb7a.png)

## Figma
![image](https://user-images.githubusercontent.com/50990925/211991680-4c74cc99-a001-4a43-9534-a62cb3d5eaaa.png)

Reply:
Fixed.